### PR TITLE
Fix "Exception when #include is not followed by a space" and "Norminette "freeze" on a C file"

### LINKED ISSUE #383: Exception when #include is not followed by a space
**Describe the bug**
When the #include is not followed by a space norminette throws an exception

**Erroneous code**
Having a test.c file with `#include<unistd.h>` throws the following error:

> Traceback (most recent call last):
>   File "/usr/local/bin/norminette", line 8, in <module>
>     sys.exit(main())
>   File "/usr/local/lib/python3.10/dist-packages/norminette/__main__.py", line 128, in main
>     registry.run(context, source)
>   File "/usr/local/lib/python3.10/dist-packages/norminette/registry.py", line 65, in run
>     ret, jump = self.run_rules(context, rule)
>   File "/usr/local/lib/python3.10/dist-packages/norminette/registry.py", line 43, in run_rules
>     self.run_rules(context, self.rules[r])
>   File "/usr/local/lib/python3.10/dist-packages/norminette/registry.py", line 32, in run_rules
>     rule.run(context)
>   File "/usr/local/lib/python3.10/dist-packages/norminette/rules/check_preprocessor_include.py", line 47, in run
>     while tkns[i].type != "NEWLINE" and i < len(tkns) - 1:
> IndexError: list index out of range

**Additional infos**
 - OS: Ubuntu 22.04.1
 - python --version: 3.10.6
 - norminette -v: 3.3.52

**Additional context**

### PR DESCRIPTION
**Pull request by LucasEGomes**
Fixes #383 and #384 
Pending more exhaustive tests.

### CODE REVIEW

File: norminette/rules/check_preprocessor_include.py [hunk 1]
```diff
@@ -44,6 +44,8 @@ def run(self, context):
                 filetype = tkns[i].value.split(".")[-1][0]
             except:
                 filetype = ""
+        if (i == len(tkns)):
```
Comment (NiumXp) [inline norminette/rules/check_preprocessor_include.py:47]: In Python, we typically do not use parentheses around the if conditions.
```suggestion
        if i == len(tkns):
```

Comment (LucasEGomes) [reply norminette/rules/check_preprocessor_include.py:47]: Sorry, it became a habit after many C projects following the norm.

**Comment by matthieu42Network** [comment]
Hello, this PR will be approved when the test are provided :)

**Comment by matthieu42Network** [comment]
Hello this has been resolved in another PR thank you

**Pull request closed without merging**